### ISSUE TITLE
problem in threecap with reactjs

### ISSUE THREAD

**Issue by eboujlal** [jbaicoianu/threecap#3]
Hello,

I am using threejs to render animation and i want to export this animation to video mp4.
When i want to use threecap library, i got an error :
TypeError: threecap__WEBPACK_IMPORTED_MODULE_7___default.a is not a constructor

And this is my code : 
```js
import THREEcap from 'threecap';


var threecap = new THREEcap({});
threecap.record({
	width: 640,
	height: 480,
	fps: 25,
	time: 10,
	format: 'mp4',
	//canvas: canvasDomElement,   // optional, slowest
	composer: this.renderer.domElement // optional, fastest
}).then(function(video) {
	video.saveFile('myVideo.mp4');
});
```

**Comment by rohan-deshpande** [comment]
This package hasn't been bundled properly for use as a module. I've just tried this myself and the only way I could get it to work was to include the lib via a script tag and access via `window.THREEcap`

**Comment by eboujlal** [comment]
Thank you i will try it

**Comment by eboujlal** [comment]
Otherwise i recorded canvas with js vanilla

**Comment by rohan-deshpande** [comment]
Yah I just published https://www.npmjs.com/package/canvas-to-video which you can try out... I haven't got any tests yet though but give it a go

**Comment by jbaicoianu** [comment]
You're right, this package was written to be used as a global script.  It should be relatively straightforward to convert this to use ES6 modules, I haven't had the time to do so myself though.
